src/Selenium2Driver.php

```diff
@@ -692,6 +690,37 @@ public function setValue($xpath, $value)
         $this->executeJsOnElement($element, $script);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function sendKeys($xpath, $value)
```

Comment:
Instead of duplication of code from “setValue” method plus few changes I’m suggesting to move shared code into new private method (or maybe protected one).